.github/workflows/linters.yml

```diff
@@ -9,28 +9,28 @@ jobs:
   lint-markdown:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - run: docker run -v $PWD:/workdir ghcr.io/igorshubovych/markdownlint-cli:latest --ignore=minutes --disable=MD013 "**/*.md"
+    - uses: actions/checkout@v3
+    - run: docker run -v $PWD:/workdir ghcr.io/igorshubovych/markdownlint-cli:latest --ignore=minutes --disable=MD013 "**/*.md"
 
-  lint-docker:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: hadolint/hadolint-action@v3.0.0
-        with:
-          recursive: true
-          ignore: DL3041
+  # lint-docker:
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - uses: actions/checkout@v3
+  #     - uses: hadolint/hadolint-action@v3.0.0
+  #       with:
+  #         recursive: true
+  #         ignore: DL3041
```

Comment:
The commit message does not say, why you comment this.

Reply:
reverted